pages/[username].js

```diff
@@ -100,28 +149,28 @@ export default function User({ users, data, BASE_URL }) {
       </Head>
 
       <Page>
-        <UserProfile data={data} BASE_URL={BASE_URL} />
+        <UserProfile data={userData} BASE_URL={BASE_URL} />
 
-        <UserTabs tabs={tabs} setTabs={setTabs} />
+        <UserTabs tabs={tabs} setTabs={setTabs} sortUserTabItems={sortUserTabItems} />
```

Comment:
Would it be better to move `sortUserTabItems` to the `components/user/UserTabs.js` file?